mux/src/tmux.rs

```diff
@@ -179,21 +144,50 @@ impl TmuxDomainState {
             *self.state.borrow_mut() = State::WaitingForResponse;
         }
     }
+
+    pub fn kick_off(domain_id: usize) {
```

Comment:
I'd suggest renaming to better reflect the purpose; adding a doc comment would be nice if things are more complicated! :-)

```suggestion
    pub fn schedule_send_next_command(domain_id: usize) {
```